### PR TITLE
PHP Conf BR 2016

### PR DESCRIPTION
**Pull request by tonysm**
### Added

- new blogpost about PHP Conf BR 2016

### CODE REVIEW

File: _posts/2016-12-12-php-conference-brasil-2016.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+He presented tools, processes and some insights on building a modern PHP application. Showed what is available for CI/CD, QA, changes in the language, optimizations, etc. Really nice, well presented talk.
+
+### API Tales of Woe and Woah! by [Phil Sturgeon](https://twitter.com/philsturgeon)
+
+Phil talked about building APIs, handling errors and shared with us his experience on building APIs, versioning and the importance of documentation. Really nice talk.
+
+### Phalcon 3.0 & Zephir by [Andrés Gutiérrez](https://twitter.com/notavecmoi)
+
+Remote talk where he presented the state of Phalcon right now (v3), how to use it as a full-stack or micro-framework and than dig in on how they built it using Zephir, a language that basically allows you to write C-extensions in code that looks a lot like PHP. Really cool stuff, I've played with Phalcon a while ago, it's looking really good these days. Zephir looks really interesting as well.
```
Comment (miggynascimento) [inline _posts/2016-12-12-php-conference-brasil-2016.md]: "and _then_ dig in"*

File: _posts/2016-12-12-php-conference-brasil-2016.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+Presented by [Elton Minetto](https://twitter.com/eminetto), a mini-tutorial convering building middlewares with [Zend Expressive](https://zendframework.github.io/zend-expressive/) and all the concepts behind micro-frameworks, middlewares and PSR-7.
+
+## Talks
+
+All the talks I attended were awesome in certain ways. I'm going to cover here the ones that I found most interesting and, again, I can only talk about the ones I attended. That being sad, I'm sorry for not covering all the talks, I'm sure they were all interesting. Also, I'm going to translate the talk names to English here, but you can check the real names and schedule on in the [conference website.](MISSING_LINK)
+
+### "How to manage quality in a project without losing your sanity" by [Elton Minetto](https://twitter.com/eminetto)
+
+A facinating topic IMHO and he shared some knowledge from the expirence he has in his own company, which made it even better.
```
Comment (miggynascimento) [inline _posts/2016-12-12-php-conference-brasil-2016.md]: experience*

File: _posts/2016-12-12-php-conference-brasil-2016.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+authors: ["tony"]
+categories: ["php", "conference", "brasil", "brazil"]
+excerpt: Last week I attended to the PHP Conference Brasil for the first time. It was awesome and I'm going to try to cover some of the best parts of it (IMHO) here.
+---
+
+Last week I attended the PHP Conference Brasil for the first time. It was awesome and I'm going to try to cover some of the best parts of it (IMHO) here.
+
+![PHPConferenceBrazil 2016](/assets/posts/2016-12-12-php-conference-brasil-2016/php-648407345584c6daaa80eb245562304.jpg)
+
+Before we start, it's worth saying that they named the conference rooms from well known community members in Brazil and there were some ElePHPant footprints guiding the way to each rooms. Really cool.
```
Comment (jdrieghe) [inline _posts/2016-12-12-php-conference-brasil-2016.md]: ... named the conference rooms *after* well known...
... the way to each *room*.

File: _posts/2016-12-12-php-conference-brasil-2016.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+![ElePHPant footprints to the rooms](/assets/posts/2016-12-12-php-conference-brasil-2016/footprints-703677630584c6daaa7aef398634759.jpg)
+
+## Tutorials
+
+First day of the conference, there were some tutorials available. I can only talk about the ones I attended, so I'm sorry that I won't cover the others.
+
+### Intelligent Bots
+
+I chose to attend the one about "_IntelligentBots_" in the first day. It was awesome. It covered building a chatbot for FB Messenger and Telegram using custom PHP code and then we used [api.ai](https://api.ai/) to build a chatbot without any line of code. Kudos to [@jacksonfdam](https://twitter.com/jacksonfdam) for this awesome tutorial and he also gave a talk on the topic, which I attended, so I'm not going to cover his talk below - but it was awesome as well.
```
Comment (hannesvdvreken) [inline _posts/2016-12-12-php-conference-brasil-2016.md]: > in the first day

On the first day I chose to attend...

File: _posts/2016-12-12-php-conference-brasil-2016.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+### Middlewares & PSR-7
+
+Presented by [Elton Minetto](https://twitter.com/eminetto), a mini-tutorial convering building middlewares with [Zend Expressive](https://zendframework.github.io/zend-expressive/) and all the concepts behind micro-frameworks, middlewares and PSR-7.
+
+## Talks
+
+All the talks I attended were awesome in certain ways. I'm going to cover here the ones that I found most interesting and, again, I can only talk about the ones I attended. That being sad, I'm sorry for not covering all the talks, I'm sure they were all interesting. Also, I'm going to translate the talk names to English here, but you can check the real names and schedule on in the [conference website.](MISSING_LINK)
```
Comment (hannesvdvreken) [inline _posts/2016-12-12-php-conference-brasil-2016.md]: > but you can check the real names

the original names (in Portugese)

Comment (hannesvdvreken) [inline _posts/2016-12-12-php-conference-brasil-2016.md]: also fix the MISSING_LINK 😂

File: _posts/2016-12-12-php-conference-brasil-2016.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+### "How to manage quality in a project without losing your sanity" by [Elton Minetto](https://twitter.com/eminetto)
+
+A facinating topic IMHO and he shared some knowledge from the experience he has in his own company, which made it even better.
+
+### "From 1 to 1.000.000 users" by [Felipe Signorini](https://twitter.com/signorini)
+
+One of the best talks about performance and scalability I've seen. He gives a really honest and sincere overview on what it takes to be able to support lots of reqs/sec. I really liked the way he presented. One of the best parts was that he said _scaling_ is not only a matter of tools, but it also requires some level of maturity of the company/team involved in the application. Another good point is that the talk was structure in such a nice way that each stage he showed what changes need to be in place to support that amount of req/sec from infrastructure to team processes (by "stages" I mean "1k req/sec", "100k req/sec", ...).
```
Comment (hannesvdvreken) [inline _posts/2016-12-12-php-conference-brasil-2016.md]: Another good point is that the talk was structure**d**

Comment (hannesvdvreken) [inline _posts/2016-12-12-php-conference-brasil-2016.md]: in such a nice way that **at** each stage he showed what

File: _posts/2016-12-12-php-conference-brasil-2016.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+### "Artificial Intelligence & PHP: pattern recognition and neural network" by [Wellton Costa](https://twitter.com/welltoncosta)
+
+Awesome talk on such an interesting topic. I'm personally facinated by this topic and I'm going to try to learn a lot more about it. He presented some examples using C++ and OpenCV, which was awesome, and then he showed what it needs to use that in PHP, compiling an extension and writing the same code in PHP. That was awesome so see in action.
```
Comment (hannesvdvreken) [inline _posts/2016-12-12-php-conference-brasil-2016.md]: Must have been a brilliant talk... but honestly Computer Vision (OpenCV) isn't the same as artificial intelligence. It's a lot of hard computational calculations and setting the right thresholds. ;-)

File: _posts/2016-12-12-php-conference-brasil-2016.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+Remote talk where he presented the state of Phalcon right now (v3), how to use it as a full-stack or micro-framework and then dig in on how they built it using Zephir, a language that basically allows you to write C-extensions in code that looks a lot like PHP. Really cool stuff, I've played with Phalcon a while ago, it's looking really good these days. Zephir looks really interesting as well.
+
+### "Full-stack: developing in the agile World" by [Rodrigo Ibraim](https://twitter.com/rodrigoibraim)
+
+He covered the changes in building web applications from the Waterfall era and now. He pointed that the term "full-stack" can be applied in any area and showed a couple of things you should know to be able to ship features and fail fast. Really nice topic.
+
+### "Real-time applications with RatchetPHP" by [Aryel Tupinambá](https://twitter.com/dfkimera)
+
+It was shown to us the concepts behind Real-time technologies and how you can build one in PHP using [RatchetPHP](http://socketo.me/) to build a WebSocket server. I've played with it before, made me want to update my own demo and maybe write a bit on it later.
```
Comment (hannesvdvreken) [inline _posts/2016-12-12-php-conference-brasil-2016.md]: > It was shown to us

is a passive sentence. You can avoid it by writing: "We learned about the concepts behind..." or "The speaker told us about the concepts behind..."

> the concepts behind Real-time technologies and how you can build one in PHP using ...

build "one real-time technology concept?
You can say:
"concepts behind Real-Time technology such as websockets, and how you can build a websocket server using..."

**Comment by hannesvdvreken** [comment]
Nice post, Tony!

**Comment by hannesvdvreken** [comment]
Yiiipiii! Our first blog post since long (second in NINE months). Thank you so much, @tonysm!

**Pull request merged**